### ISSUE TITLE
Black overlay destroys gradient

### ISSUE THREAD

**Issue by pasomnica** [MohammadAG/Xposed-Tinted-Status-Bar#101]
Build was compiled today. This issue is affecting mostly Nova Launcher, where original gradient status bar (660000) isn't working, rather is replaced with 20% black overlay. Same goes for one of the music player (My Cloud Player) where gradient is working similar like in Nova (glass background). Both issues is present on 4.1.2 JB. I can't test KK or another one. Attached screenshot of Nova bug and My Cloud Player in original state and bug state.
![screenshot_2014-08-21-14-47-39](https://cloud.githubusercontent.com/assets/7683736/3996170/350bca54-2932-11e4-864f-da3ea31f9dbe.png)

![screenshot_2014-06-07-19-26-44](https://cloud.githubusercontent.com/assets/7683736/3996161/10826490-2932-11e4-9f23-5c14480addfe.png)

![screenshot_2014-08-21-14-49-27](https://cloud.githubusercontent.com/assets/7683736/3996182/64de095e-2932-11e4-9178-66dd31b6e30a.png)

Long story short, is there chance to set gradient status bar for some apps even when black overlay is choosen by default?
Thanks in advance.
